cms/envs/labster.py

```diff
@@ -19,3 +19,8 @@
 )
 
 LABSTER_WIKI_LINK = LABSTER_SETTINGS.get('LABSTER_WIKI_LINK', 'https://theory.labster.com/')
+
+# Sentry integration config
+RAVEN_CONFIG = AUTH_TOKENS.get('RAVEN_CONFIG', {})
+if RAVEN_CONFIG['dsn']:
```

Comment:
you will get KeyError exception if `RAVEN_CONFIG` is not set. Use `RAVEN_CONFIG.get('dsn')`